components/ca-form.js

```diff
@@ -1094,6 +1094,8 @@ define([
             if (format === 'radio') {
 
                 el = createElementLegacy(null, 'ca-radio-group', {});
+                el.data = schema.extends[0];
```

Comment:
Yup, all good points.  I don't think it'll take long to make the necessary adjustments, especially as I have template code already which I'll make shared.  A few tweaks to the tests and it'll be ready to drop in I reckon.